pkg/api/v1/types.go

```diff
@@ -1258,6 +1258,19 @@ type PodSpec struct {
 
 // PodSecurityContext holds pod-level security attributes and common container settings.
 type PodSecurityContext struct {
+	// SupplementalGroups can be used to specify a list of
+	// additional groups which the main container process will run
```

Comment:
K, I will make an edit in this PR, but that message is actually from #14991 which this depends on.